docs/faq/index.rst

```diff
@@ -1,17 +1,276 @@
+You can use NFS and block storage side by side, they are not exclusive.
+
+Can I attach a single volume to multiple hosts?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Not currently, support from multi-attach backends like `GCE in Read Only mode`, or NFS-like backends like storage, or distributed filesystems like GlusterFS would need to be integrated.
+Flocker focuses mainly on block-storage uses cases that attach a volume to a single node at a time.
+
+What happens when the node where my container is running dies, crashes or restarts?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Flocker will make sure the volume is detached, re-attached and mounted to the correct node when the container starts on a new healthy node.
+You will not have to manage these operations manually.
+
+Does Flocker Control support HA?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+No, the control service is not HA and has no integration with key values storage, though it is on our future roadmap.
+
+The control service can stop and all containers and the dataset will continue to function.
+
+If you want to provide sudo-HA for the control service you can.
+The control service saves a :file:`.json` file in :file:`/var/lib/flocker/`, of which you can take periodic backups.
+In case of failure you can easily restore the control service, as long as you replace the :file:`.json` file and have all the needed certificates that were used by the failed control service.
+
+Backing up your control and cluster certificates along with the :file:`.json` file is a good idea.
+
+Are AWS IAM roles supports with AWS?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Not currently, our AWS backend just uses ``ACCESS KEY/ ACCESS TOKEN``
+
+Are volume snapshots supported with Flocker?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Not yet.
+
+What happened to ZFS support?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+ZFS actually moves data bits around your data center when you need volumes to move; because of this it was inherently slower for the use-cases Flocker is tackling.
+
+Flocker can do everything and more than it did with ZFS with its current supported backends.
+ZFS also has a number of inefficiencies that lead us to move away from support for ZFS and focus on the other backends that we currently support.
+
+We are excited that ZFS was adopted by Canonical/Ubuntu, and we understand some people liked ZFS, but at this moment it is not supported.
+That being said, if we see or hear reasons to support it again, we are always open to having the conversations of why you think we should, so feel free to reach out.
+
+Will you support Nomad, RancherOS, etc?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If we find that enough users want support for Flocker in other frameworks or projects then we will certainly consider it.
+We have looked at adding support to Rancher in the past but it’s not currently something we are working on.
```

Comment:
This section could also suggest that people can contribute support if they want to.